demos/src/deploy-solution-main.ts

```diff
@@ -107,15 +110,19 @@ export function deploySolution(
   srcAuthentication: common.UserSession,
   destAuthentication: common.UserSession,
   progressCallback: common.ISolutionProgressCallback,
-  enableItemReuse: boolean
+  enableItemReuse: boolean,
+  customParams: any
 ): Promise<string> {
   // Deploy a solution described by the supplied id
   const options: common.IDeploySolutionOptions = {
     jobId,
     progressCallback: progressCallback,
     consoleProgress: true,
     storageAuthentication: srcAuthentication,
-    enableItemReuse
+    enableItemReuse,
+    templateDictionary: {
+      params: JSON.parse(customParams)
```

Comment:
If a folder of solution templates is supplied and the Custom Params input box is left alone, "Deploying..." appears on the screen and then is replaced by "{}".